src/traits.rs

```diff
@@ -1,6 +1,32 @@
 use crate::{IsoWeek, Weekday};
 
 /// The common set of methods for date component.
+///
+/// Methods such as [`year`], [`month`], [`day`] and [`weekday`] can be used to get basic
+/// information about the date.
+///
+/// The `with-*` methods can change the date.
```

Comment:
Let's use `with_` consistently instead of sometimes using `with-`.

Reply:
Oops, that was a `-` by mistake.